src/lib/config.rs

```diff
@@ -1,6 +1,7 @@
 use std::fs;
 
 use serde_derive::Deserialize;
+use toml_edit::{value, Document};
```

Comment:
I am a bit skeptical about using this library for editing config, as it is not very popular. Can we look for some alternative for this, like parsing config.toml first to the struct then setting the required value and updating the file finally.

Reply:
This package is itself based on `toml` package. I am using this particular one because this preserves the comments. The normal approach would delete them